### PR TITLE
Aktualizr: backport main-next fixes

### PR DESCRIPTION
**Pull request by quaresmajose**

### CODE REVIEW

**Comment by ricardosalveti** [comment]
Do we want to wait for the aktualizr patches to be merged in our tree first or is there a need to land this now?

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend [hunk 1]
```diff
@@ -19,8 +19,6 @@ PACKAGECONFIG[ubootenv] = ",,u-boot-fw-utils,u-boot-fw-utils u-boot-default-env
 PACKAGECONFIG[libfyaml] = ",,,libfyaml"
 PACKAGECONFIG[aklite-offline] = "-DBUILD_AKLITE_OFFLINE=ON,-DBUILD_AKLITE_OFFLINE=OFF,"
 
-PKCS11_ENGINE_PATH = "${libdir}/engines-3/pkcs11.so"
```
Comment (mike-sul) [inline meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:22]: Why do we revert it? Is it because it was added in `meta-updater`?

Comment (ricardosalveti) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:22]: Exactly.

**Comment by mike-sul** [comment]
> Do we want to wait for the aktualizr patches to be merged in our tree first

The patches have been already merged into aklite's master, so I think we can just bump new aklite version instead of patching.

**Comment by ricardosalveti** [comment]
> > Do we want to wait for the aktualizr patches to be merged in our tree first
> 
> The patches have been already merged into aklite's master, so I think we can just bump new aklite version instead of patching.

Great, that is indeed better.

File: meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend (hunk 1 above)
Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/aktualizr/aktualizr_%.bbappend:22]: yes the patch is already merged on `meta-updater`

**Comment by quaresmajose** [comment]
Ok so I will wait for the bump of the aktualizr and rebase after, doping the [base: aktualizr: fix compilation](https://github.com/foundriesio/meta-lmp/pull/800/commits/843db91f99819733efa9a8b91be6f8b9d1064d90)

**Comment by ricardosalveti** [comment]
> Ok so I will wait for the bump of the aktualizr and rebase after, doping the [base: aktualizr: fix compilation](https://github.com/foundriesio/meta-lmp/pull/800/commits/843db91f99819733efa9a8b91be6f8b9d1064d90)

Just do the dump as part of this pr.

**Review by mike-sul — APPROVED** [review]

**Review by angolini — APPROVED** [review]

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**